dlp/snippets/deid/deid_table_with_multiple_crypto_hash.go

```diff
@@ -0,0 +1,202 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package deid
+
+// [START dlp_deidentify_table_with_multiple_crypto_hash]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	dlp "cloud.google.com/go/dlp/apiv2"
+	"cloud.google.com/go/dlp/apiv2/dlppb"
+)
+
+// deIdentifyTableWithMultipleCryptoHash Transforms findings using two separate cryptographic hash transformations.
+func deIdentifyTableWithMultipleCryptoHash(w io.Writer, projectID, transientKey1, transientKey2 string) error {
```

Comment:
please use transientKeyName1, transientKeyName2

Reply:
Updated it.